src/pages/index.tsx

```diff
@@ -1,8 +1,16 @@
 import type { NextPage } from "next";
 import { DialogTitle } from "@mui/material";
+import Link from "next/link";
 
 const Home: NextPage = () => {
-  return <DialogTitle>This is where we start to develop our frontend</DialogTitle>;
+  return (
+    <DialogTitle>
+      This is where we start to develop our frontend.
+      <Link href="/signup">
```

Comment:
What is this for? I think we already have a signup page

Reply:
This is just a temporary link to sign up page, because the default page of localhost:3000 is this index page, I add a like to sign up page, so we can easy to go to sign up page.